docs/contracts/v4/concepts/02-pool-manager-and-pool.mdx

```diff
@@ -0,0 +1,175 @@
+---
+id: pool-manager-and-pool-initialization
+title: Pool Manager and Pool Initialization
+sidebar_position: 1
+---
+
+# Architecture
+In Uniswap v3, each pool has its own contract instance, which makes making pools and doing swaps in multiple pools costly
```

Comment:
While I think a v3-vs-v4 is valuable in the right circumstances, I'm beginning to worry that it adds cognitive load to the users.

My concern is that new devs may feel like they need to understand v3 before going into v4